tools/scripts/upgrades/start-node.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash  
```

Comment:
#PUC a bit more on where/why this is necessary.

Reply:
Removed the file as it was repeating the docs.